### ISSUE TITLE
Error creating nar with scoped packages

### ISSUE THREAD

**Issue by aledbf** [h2non/nar#107]
After an upgrade to `npm` 2.1.16 and the use of scoped packages I get this:
```
$ nar create
Error: ENOENT, open '/var/folders/my/b3809nt94lz0y420dtd2cp0m0000gn/T/nar-app-1420604772482/@private/web.tar'
```

nar 0.3.15
node 10.0.35

Any idea?
Thanks

**Comment by h2non** [comment]
Could you share the `package.json` dependencies field?

**Comment by h2non** [comment]
The issue basically exists because the new `npm` scope expression is treated as file path in nar.
It will be fixed in `0.3.16` release, hopefully today.

**Comment by aledbf** [comment]
This reproduces the error:

package.json
```
{
  "name": "scope-error",
  "version": "1.0.0",
  "main": "index.js",
  "dependencies": {
    "@private/request": "^2.51.0"
  }
}
```

```
npm i request
sed -i back 's/"name": "request"/"name": "@private\/request"/' ./node_modules/request/package.json
mkdir node_modules/@private
mv node_modules/request node_modules/@private
echo "module.exports = {}" > index.js
```

```
nar create -d -v
Creating archive...
Error: ENOENT, open '/var/folders/my/b3809nt94lz0y420dtd2cp0m0000gn/T/nar-scope-error-1420630363945/@private/request.tar'

```

**Comment by h2non** [comment]
Thanks for reporting.
This night I will release the fix and the new version.

**Comment by h2non** [comment]
Fixed in `0.3.16` and added [multiple](https://github.com/h2non/nar/blob/master/test/create.ls#L108) [tests](https://github.com/h2non/nar/blob/master/test/nar.ls#L218) to cover that functionality.

Simply upgrade it: `npm update -g nar`

**Comment by aledbf** [comment]
@h2non thanks
